### PR TITLE
Added constants throughout codebase and reformatted code

### PR DESCRIPTION
**Pull request by debangeedas**
… codestyle (addressing issue #8)

### CODE REVIEW

**Comment by debangeedas** [comment]
Hey! I ran the build on my local machine but it didn't fail. However, the check is failing on github. Could you please suggest any solution?

**Review by JohnathonNow — COMMENTED** [review]
Did you commit your new Constants file? I'm not seeing it here / on your branch it seems identical to main.

File: src/main/java/com/johnwesthoff/bending/logic/World.java [hunk 1]
```diff
@@ -28,60 +13,51 @@
...
     public float jump = 0;
     public boolean MB1, MB2, MB3;
-    public int landTexSize = 256;
-    public int FPS = 0;
-    public static int head = 26;
-    public static int body = 13;
-    public int flowCount = 0, maxFlow = 5000;
+
     public TexturePaint skyPaint, grassPaint, sandPaint, stonePaint, barkPaint, icePaint, nightPaint;
-    public static final byte AIR = 0, GROUND = 1, WATER = 2, OIL = 3, LAVA = 4, SAND = 5, STONE = 6, TREE = 7, ICE = 8,
```
Comment (JohnathonNow) [inline src/main/java/com/johnwesthoff/bending/logic/World.java:73]: It looks like these were moved to Constants, yet within this file they don't seem to reference those.

**Comment by debangeedas** [comment]
I have not created a new file, I have added to the existing constants file.

> Did you commit your new Constants file? I'm not seeing it here / on your branch it seems identical to main.

File: src/main/java/com/johnwesthoff/bending/logic/World.java (hunk 1 above)
Comment (debangeedas) [reply src/main/java/com/johnwesthoff/bending/logic/World.java:73]: Yes they are being referenced from the Constants.java file. Do take a look at the imports.

Comment (JohnathonNow) [reply src/main/java/com/johnwesthoff/bending/logic/World.java:73]: Ah I see now, thanks.

**Comment by JohnathonNow** [comment]
> I have not created a new file, I have added to the existing constants file.
> 
> > Did you commit your new Constants file? I'm not seeing it here / on your branch it seems identical to main.

I meant did you commit your changes to that file, as it appears identical to the one on upstream.

**Comment by debangeedas** [comment]
> > I have not created a new file, I have added to the existing constants file.
> > > Did you commit your new Constants file? I'm not seeing it here / on your branch it seems identical to main.
> 
> I meant did you commit your changes to that file, as it appears identical to the one on upstream.

Yeah... I did commit all the changes. You can see them right? In my pull request does it not show 81 files changed?

**Comment by JohnathonNow** [comment]
> > > I have not created a new file, I have added to the existing constants file.
> > > > Did you commit your new Constants file? I'm not seeing it here / on your branch it seems identical to main.
> > 
> > 
> > I meant did you commit your changes to that file, as it appears identical to the one on upstream.
> 
> Yeah... I did commit all the changes. You can see them right? In my pull request does it not show 81 files changed?

Here is your Constants file, copied from your branch:
https://github.com/debangeedas/Bending/blob/b312f5be419f382285d6d522ef16ee1aaf924469/src/main/java/com/johnwesthoff/bending/Constants.java#L1-L11

Here is the one currently on my branch:
https://github.com/JohnathonNow/Bending/blob/23f4ecf98c5258b9592b2a5fc94ee2463a69dc4a/src/main/java/com/johnwesthoff/bending/Constants.java#L1-L11

**Comment by debangeedas** [comment]
How is this possible... Give me 5 minutes i will commit again

**Comment by debangeedas** [comment]
Can you see the changes in the latest commit?

**Comment by JohnathonNow** [comment]
Yes I can, thanks. I can take care of the merge conflicts.

**Comment by debangeedas** [comment]
Thanks a lot!

**Comment by debangeedas** [comment]
Hey! Is there any update?

**Comment by debangeedas** [comment]
I hate to be a bother, but I would really appreciate any update on the status

**Comment by JohnathonNow** [comment]
Sorry, I should have a chance to merge later today.

**Pull request merged**